src/model/script_config.py

```diff
@@ -27,7 +27,16 @@ def __init__(self):
         self.allowed_users = []
         self.admin_users = []
         self.group = None
-
+        self.parsing_failed = False
+
+def create_failed_ShortConfig(path):
+    failed_short_config = ShortConfig()
+    failed_short_config.name = path
+    with open(path) as f:
+      if '"allowed_users"' in f.read():
+          failed_short_config.name = 'file with possibly restricted access'
```

Comment:
I'm not sure that having 2 configs with the same name would work fine for script server.

Reply:
Checked, it really causes some problems. Added path hash.